DNN Platform/Library/Entities/Urls/AdvancedUrlRewriter.cs

```diff
@@ -1533,6 +1533,10 @@ private static bool CheckForDebug(HttpRequest request, NameValueCollection query
                                              portalSettings.ActiveTab.TabName + " IsSecure: " +
                                              portalSettings.ActiveTab.IsSecure.ToString());
 
+                    // this covers the specific case of popups and iframes opened from a secure tab
+                    // e.g: AdminExperience -> Users -> UsersProfile 
+                    // example url: https://YOUR_HOST/Admin/User-Accounts/ctl/Edit/mid/464/UserId/4/editprofile/true?popUp=true
+                    var isPopUpFromSecurePage = result.IsSecureConnection && UrlUtils.IsPopUp(url);

```

Comment:
Can we confirm, or document here what the Result & Url values are?  It appears from this that the URL is the destination, Result is current page?